Resources/Textures/Clothing/Neck/Cloaks/Synd.rsi/Prototype.txt

```diff
@@ -0,0 +1,10 @@
+- type: entity
+  parent: ClothingNeckBase
+  id: ClothingNeckCloakSynd
+  name: Syndicate cloak
+  description: dark like a crushed station. Nicely!
+  components:
+  - type: Sprite
+    sprite: Clothing/Neck/Cloaks/SyndiCloak.rsi
+  - type: StealTarget
+    stealGroup: SyndiCloak
```

Comment:
Do not put an entity prototype in Resources/Textures, and do not put yaml code in a .txt file

This should go somewhere in /home/useriel/s14/Resources/Prototypes/Entities/Clothing/Neck/
Probably cloaks.yml

Reply:
Should this be a Steal Target if its not spawning anywhere on the station?

Reply:
Sorry. I have only recently started to understand this matter, and so far I lack experience...